plugins/systemstat.koplugin/main.lua

```diff
@@ -44,8 +44,16 @@ function SystemStat:appendCounters()
     if self.resume_sec then
         self:put({_("  Last resume time"), os.date("%c", self.resume_sec)})
     end
-    self:put({_("  Up hours"),
-             string.format("%.2f", os.difftime(os.time(), self.start_sec) / 60 / 60)})
+    self:put({_("  Up time"),
```

Comment:
Could you also fix this.

Reply:
I don't quite understand what to fix here (and in the other two comments). "hours" seem wrong to me, as we show a duration "01:20:30".

Reply:
Translators shouldn't be bothered with this indentation stuff. :-)

```suggestion
    self:put({"  " ..  _("Up time"),
```